config/config.properties

```diff
@@ -183,16 +183,16 @@ metrics.enableStatistics=false
 # Valid values: write-accessible filesystem directory.
 #metrics.csvEndpoint=/tmp/csv
 
-# Whether to enable StatsD reporting.
+# Whether to enable Statsd reporting.
 # Valid values: 'true' or 'false'.
 #metrics.enableStatsdReporter=true
 
-# StatsD endpoint host name.
-#metrics.statsDHost=localhost
+# Statsd endpoint host name.
+#metrics.statsdHost=localhost
```

Comment:
I think that changing this may have consequences in any application passing parameters to Pravega, right? Please, check with platform team if this is working after the change.

Reply:
Co
uld you elaborate on why you think that is?

Reply:
Not answering on Raul's behalf - The Platform team will make certain config changes to adopt this migration, e.g. start using InfluxDB directly, and provide influxdb connection details through config. 
(Not sure if the original question is about backwards compatibility, but config isn't within the scope I think)

Reply:
Just thinking in scripts, applications, etc. already using the previous name (`statsDHost`), they may need to be aware of this change for configuring the StatsD reporter correctly. If this situation is already controller, that's great.

Reply:
Yes the Platform team uses statsDXXX.